### ISSUE TITLE
ConPTY: Just delete a character instead of reprinting a line when using backspace on a prompt

### ISSUE THREAD

**Issue by Tyriar** [microsoft/terminal#1646]
# Environment

```none
Windows build number: Microsoft Windows [Version 10.0.18922.1000]
vscode 1.36.0 insiders
```

# Steps to reproduce

1. Open terminal
2. Press a
3. Press backspace

# Expected behavior

This is what happens when on Linux:

```
a
[K
```

# Actual behavior

```
a[0K
[?25l
C:\Users\daimms.REDMOND>[0K[?25h
```

# Why?

The incoming data is given to the screen reader to announce, so whenever backspace is pressed it would read out the prompt again.

**Comment by zadjii-msft** [comment]
(@Tyriar and the same with this one too)

**Comment by Tyriar** [comment]
Seems fixed on 10.0.18970.1005:

![image](https://user-images.githubusercontent.com/2193314/64541503-d337e200-d2d6-11e9-9aeb-4e0a759825c5.png)
